tests/unit/test-util/test.updates.js

```diff
@@ -18,12 +17,18 @@ describe('util/updates', () => {
         version: '1.0.0',
         updateNotifier: updateNotifierStub,
       });
-      assert.equal(updateNotifierStub.called, true);
-      assert.equal(updateNotifierStub.firstCall.args[0].pkg.name, 'web-ext');
-      assert.equal(updateNotifierStub.firstCall.args[0].pkg.version, '1.0.0');
-      assert.isNumber(updateNotifierStub.firstCall.args[0].updateCheckInterval);
-      assert.equal(updateNotifierStub.firstCall.args[0].updateCheckInterval,
-                   1000 * 60 * 60 * 24 * 3);
+
+      sinon.assert.calledWith(
```

Comment:
I think it would be better to remove this assertion because the one below it checks `updateCheckInterval` in a more precise manner.